k8s/helmfile/env/production/queryservice-gateway.values.yaml.gotmpl

```diff
@@ -2,14 +2,14 @@ platform:
   apiBackendHost: api-app-backend.default.svc.cluster.local
 
 image:
-  tag: "2.1"
+  tag: "2.2"
   repository: ghcr.io/wbstack/queryservice-gateway
   pullPolicy: Always
 
 resources:
   requests:
-    cpu: 60m
-    memory: 50Mi
+    cpu: 125m
```

Comment:
Not sure if these changes are really needed. The service might do much more work now though, so this is mostly being cautious or something similar.

Reply:
I expect this to easily fit so no objection to bumping it